app/concerns/liquid_interpolatable.rb

```diff
@@ -386,21 +386,27 @@ def initialize(tag_name, markup, tokens)
         else
           raise Liquid::SyntaxError, 'Syntax Error in regex_replace tag - Valid syntax: regex_replace pattern in'
         end
-        @nodelist = @in_block = []
-        @with_block = nil
```

Comment:
I think it's better not to remove this initialization, because it was there to avoid a runtime warning about reference to an uninitialized instance variable when `$VERBOSE` is turned on.